### PR TITLE
Added IC 7485 and 4 analog functions

### PR DESCRIPTION
**Pull request by piymis**
I have added one 7485 IC ( at the bottom of 7400 IC family script) .All the functioning logic of IC is correct , only issue is on output pin 5 and 7 the ORing operation using the built in OR gates raises an error

I have also added short handy functions in Algorithm / AnalogFormulas script to calculate the series and parallel resistance and capacitance

### CODE REVIEW

File: BinPy/Algorithms/AnalogFormulas.py [hunk 1]
```diff
@@ -103,3 +103,34 @@ def evaluate(self, i=None, v=None, z=None, p=None, c=None):
...
             return {'i': i, 'v': v, 'z': z, 'p': p, 'c': c}
+
+# Handy Functions to calculate the net total series and parallel resistance and capacitance of two 
+# individual resistances and capacitances			
+def SeriesR(a,b):
+	try:
+		total=float(a)+ float(b)
+		return total
+	except ValueError:
+		print "Error: Please enter numeric values"
```
Comment (debugger22) [inline BinPy/Algorithms/AnalogFormulas.py:114]: @piymis To make it compatible with Python3 you should use new print function. 
I think raising an error is better here than just printing it so you should change it to `raise ValueError("Please enter numeric values")`

**Comment by debugger22** [comment]
You should also add tests for the functions you've added.

**Comment by piymis** [comment]
Thanks for your review.I will incorporate these things.

**Comment by debugger22** [comment]
Closing it for now. Please reopen it if you want to continue.

**Pull request closed without merging**